src/libs/terms-of-service-alerts.js

```diff
@@ -31,7 +31,7 @@ const getNewTermsOfServiceNeedsAcceptingAlert = async termsOfServiceState => {
     title: 'There is a new Terra Terms of Service.',
     message: h(Fragment, [
       h(Fragment, { key: 'customText' }, [gracePeriodText]),
-      h(Fragment, { key: 'lineBreak' }, [<br/>]),
+      h(Fragment, { key: 'lineBreak' }, [br()]),
```

Comment:
Fails to build with Esbuild.

Reply:
🤔 I didn't find something in a quick search, but I wonder if there's some ESLint configuration to disallow JSX.

Reply:
Aha, found a way to do this. https://github.com/DataBiosphere/terra-ui/pull/3684

Reply:
I was going to suggest a follow-up PR. Nicely done.